prow/plank/controller.go

```diff
@@ -180,6 +181,12 @@ func (c *Controller) syncJenkinsJob(pj kube.ProwJob) error {
 	if pj.Complete() {
 		return nil
 	} else if pj.Status.State == kube.TriggeredState {
+		if pj.Spec.MaxConcurrency > 0 && len(pjm[pj.Spec.Job]) >= pj.Spec.MaxConcurrency {
+			// do not start more jobs than allowed
+			// TODO(nclandolfi): log?
+			return nil
+		}
+
 		// Start the Jenkins job.
 		pj.Status.State = kube.PendingState
```

Comment:
I may misunderstand the states.

I thought that Pending meant that the job had been started, i.e., was currently running, and triggered just meant it was "queued." Because we don't want to start the job we want to leave it in the triggered state, right? And then not include those jobs in the list of jobs which are pending...